b9lab-content/5-my-own-chain/03-starport-16-wager-denom.md

```diff
@@ -0,0 +1,82 @@
+---
+title: IBC Foreign Token Wager
+order: 18
+description: You let players wager any fungible token.
+---
+
+# IBC Foreign Token Wager
+
+When you introduced a wager, players could play a game and wager the base, staking, token of your blockchain application. What if they want to play with other _currencies_? Fortunately, with the use of IBC, your blockchain can represent a token from any other blockchain connected to yours. Here, you are agnostic as to which tokens are being represented, and who is taking care of relayers. Instead, your only concern is to enable the use of _foreign_ tokens.
+
+## New Information
+
+This is extremely simple. Instead of defaulting to `"stake"`, you now let players decide what string their token is. So you update:
+
+* The stored game, in `proto/checkers/stored_game.proto`:
+    ```proto [https://github.com/cosmos/b9-checkers-academy-draft/blob/9799e2cee1a0541932ec19d5cfdcdd955be0390f/proto/checkers/stored_game.proto#L21]
+    message StoredGame {
+        ...
+        string token = 13; // Denomination of the wager.
+    }
+    ```
+* The message to create a game, in `proto/checkers/tx.proto`:
+    ```proto [https://github.com/cosmos/b9-checkers-academy-draft/blob/9799e2cee1a0541932ec19d5cfdcdd955be0390f/proto/checkers/tx.proto#L46]
+    message MsgCreateGame {
+        ...
+        string token = 5; // Denomination of the wager.
+    }
+    ```
+
+To have Starport and Protobuf recompile both files, you can use:
+
+```sh
+$ starport chain serve
```

Comment:
since code fences show the right display, I recommend we omit the symbol that indicates you are at the command line, from long experience I know users like to copy and paste the commands

```suggestion
starport chain serve
```

Reply:
@xavierlepretre I think this is a good idea.

Reply:
If I remove the `$`, I would adjust by writing "in a shell" in most or all the sentences that introduce the commands.